README.md

```diff
@@ -1 +1,64 @@
-# metrics-app
\ No newline at end of file
+# metrics-app
+
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
+
+> Mobile web application builded with React and Redux.
+
+![home](https://user-images.githubusercontent.com/9049260/142418227-e5946248-2607-4aa6-9022-ed873d08367b.png)
+![details](https://user-images.githubusercontent.com/9049260/142418311-17783d8d-d7b6-4d01-8ff6-344944bca413.png)
+
+This app can be used to check the numeric values of COVID cases in various European countries.
+## Built With
+
+- React & Redux
+- JSX
+- CSS
+
+## Live Demo
+
+- [Heroku](https://metrics-app-covid.herokuapp.com/)
+
+## Presentation Video
+
+- [Link](https://youtu.be/8kpV2FQUWKQ)
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+    Node >= 14.0.0 and npm >= 5.6
+
+### Setup
+
+    1. git@github.com:ntzwilly/metrics-app.git
+    2. cd metrics-app
+
+### Install
+
+    npm install
+
+### Usage
+
+    npm start
+
+## Authors
```

Comment:
- [ ] To improve the professionalism aspect of your ReadMe, kindly change the header title _Authors_ to _Author_, as this project is done solely.

Reply:
Thank you. I have made the change

Reply:
This change has been done well